client/src/pages/TaggingPage.js

```diff
@@ -1,21 +1,69 @@
 import React, { useState } from "react";
-import { Button } from "../components/Button";
+import { useEffect } from "react";
 import styled from "styled-components/macro";
+import { Button } from "../components/Button";
 import { addNewTag } from "../utils/api";
+import { getImageObj } from "../utils/api";
+import useAsync from "../utils/useAsync";
 
 const Display = styled.div`
   border: solid 1px lightgray;
+  display: flex;
+  justify-content: center;
+`;
+const ImageContainer = styled.div`
+  height: 800px;
+  width: 800px;
+  border: solid 1px lightgray;
+`;
+const ImageSlide = styled.div`
+  border: solid 1px lightgray;
+  display: flex;
+  justify-content: center;
+  flex-direction: row-reverse;
+`;
+const Thumbnail = styled.img`
+  width: 10vw;
+  margin: 0 0.5rem;
 `;
 const TagForm = styled.form`
   border: solid 1px lightgray;
 `;
+const TagNotifier = styled.div`
+  border: solid 1px lightgray;
+`;
```

Comment:
DRY ;)

Reply:
Thanks! I will put this into a variable when working on the style.